Git/GitHub.Common/Clients/GitHubClient.cs

```diff
@@ -40,13 +40,13 @@ public GitHubClient(string apiBaseUrl, string userName, SecureString password, s
 
         public async Task<IList<Dictionary<string, object>>> GetRepositoriesAsync()
         {
-            UriBuilder url;
+            string url;
             if (!string.IsNullOrEmpty(this.OrganizationName))
-                url = new UriBuilder($"{this.apiBaseUrl}/orgs/{Uri.EscapeUriString(this.OrganizationName)}/repos?per_page=100");
+                url = $"{this.apiBaseUrl}/users/{Uri.EscapeUriString(this.OrganizationName)}/repos?per_page=100";
```

Comment:
Was the change from /orgs/ to /users/ here intentional?

Reply:
Yes - https://api.github.com/users/Inedo/repos and https://api.github.com/orgs/Inedo/repos are identical, but https://api.github.com/orgs/BenLubar/repos gives an error whereas https://api.github.com/users/BenLubar/repos does not. This is because organizations are a subtype of users.

Reply:
Figured it was something like that, but just wanted to make sure.